pkgs/top-level/release.nix

```
@@ -11,8 +11,8 @@
 { nixpkgs ? { outPath = (import ../../lib).cleanSource ../..; revCount = 1234; shortRev = "abcdef"; revision = "0000000000000000000000000000000000000000"; }
 , officialRelease ? false
   # The platforms for which we build Nixpkgs.
-, supportedSystems ? [ "x86_64-linux" "x86_64-darwin" "aarch64-linux" "aarch64-darwin" ]
-, limitedSupportedSystems ? [ "i686-linux" ]
+, supportedSystems ? [ "x86_64-unknown-linux-gnu" "x86_64-apple-darwin" "aarch64-unknown-linux-gnu" "aarch64-apple-darwin" ]
+, limitedSupportedSystems ? [ "aarch64-unknown-linux-musl" "i686-unknown-linux-gnu" "x86_64-unknown-linux-musl" ]
```

```suggestion
, supportedConfigs ? [ "x86_64-unknown-linux-gnu" "x86_64-apple-darwin" "aarch64-unknown-linux-gnu" "aarch64-apple-darwin" ]
, limitedSupportedConfigs ? [ "aarch64-unknown-linux-musl" "i686-unknown-linux-gnu" "x86_64-unknown-linux-musl" ]
```
I wonder if this helps or hurts

---

Hmm, yeah, we're expecting a different format, so renaming the parameters (hence making it visible to callers that something has changed) makes sense.

---

It actually looks like there are callers of this file that I forgot to update, so I'll fix that too.

---

Not in many cases, I think.  At hydra.nixos.org instance at least :-)

---

I've had a go at the first one, but it's too much work.  Maybe one day.  It would also have some other drawbacks, like renaming every single Hydra job.  I'll see how feasible the second one is.

---

Ah right, renaming jobs would be unpleasant.

---

I don't mind the extra work part :P, but yes I suppose someone could get made if all the hydra job names start changing.

---

(No longer relevant to the new attempt.)